_posts/2020-09-07-introduction-to-system-tests.md

```diff
@@ -0,0 +1,275 @@
+---
+layout: post
+title:  "Introduction to system tests"
+date: 2020-09-07
+author: maros_orsak
+---
+
+Testing is an essential part of the software cycle.
+Failures occur when testing is neglected.
+In Strimzi, we are well aware of the need to test features.
+Follow me on this blog post to see a basic view of our tests.
+
+In this post we'll look at system tests and get **you** motivated to do the same.
+
+## Motivation
+
+Maybe you've been asking yourself the following questions:
+
+- Why do we need to write system tests?
+- Is there any value to system tests?
+- Do they catch errors that unit or integration tests don't?
+
+The answer is very simple.
+Yes!
+System tests are a key part of being able to ship Strimzi to users with more confidence that it will not crash in production.
+The types of tests that validate the basic properties of a system we call [_Smoke tests_](https://www.guru99.com/smoke-testing.html)
+Smoke tests detect major issues, such as issues with the deployment of an application or general configuration.
+Moreover, we have regression test suites. 
+Regression tests cover most features and will definitely find a bug if one is caused by recent changes in verified code.
```

Comment:
I wouldn't use "definitely". It's software, it _wants_ to be buggy, and no amount of testing will catch all the bugs.